### ISSUE TITLE
Tab key to move to the next form element ignoring checkboxes and radio buttons

### ISSUE THREAD

**Issue by sukhwinder33445** [Icinga/icingaweb2#4411]
## Describe the bug
It is possible to set the focus to the next form element by pressing the tab key. Unfortunately, the tab key completely ignores the checkboxes and radio buttons.

SAFARI : 
* Ignore the Login button on the login page.
* Ignore main menu items and submit buttons.

The main menu items can be selected with the tab key, but are not really visible to the user. As can be seen in the screenshot, you can only see in the link which menu item is currently selected with the tab key.

![Screenshot 2021-07-01 at 09 17 22](https://user-images.githubusercontent.com/54990055/124083227-ec7ec500-da4d-11eb-9f12-56a62ec93afd.png)

## To Reproduce
* Login page: try to fill in without using the mouse and checking the stay logged in checkbox.
* Account preferences: Try to change theme mode or deactivate the "Show Stacktraces" checkbox with the Tab key.

**Comment by nilmerg** [comment]
F**, that's due to the fix for #4310 :roll_eyes:

**Comment by sukhwinder33445** [comment]
I checked out to the branch `disable-dark-theme`. Here you have the same problems as described above + the tabs are not readable because of the white background and the white font colour when you move over the tabs with the tab key. 

![Screenshot 2021-07-01 at 10 30 20](https://user-images.githubusercontent.com/54990055/124094120-c3affd00-da58-11eb-8923-9b78f860f57d.png)

**Comment by nilmerg** [comment]
This and

> SAFARI :
>
> Ignore the Login button on the login page.
> Ignore main menu items and submit buttons.
> The main menu items can be selected with the tab key, but are not really visible to the user. As can be seen in the screenshot, you can only see in the link which menu item is currently selected with the tab key.

are different issues. Please create separate issues for them.

**Comment by nilmerg** [comment]
Issues you have with `disable-dark-theme` please write them into the PR.
